packages/corvid-cli/test/edit.e2e.js

```diff
@@ -37,7 +37,7 @@ describe("edit", () => {
         ".corvidrc.json": '{ "metasiteId": "12345678" }',
         src: {
           backend: {},
-          frontend: {},
+          assets: {},
```

Comment:
i don't think those subfolders are necessary.
check the init site method we're using for the server tests (and maybe start using it here)